tests/foreman/ui/test_registration.py

```diff
@@ -204,6 +206,7 @@ def test_positive_global_registration_end_to_end(
     ]
     for pair in expected_pairs:
         assert pair in cmd
+
```

Comment:
Do we need proxy configuration here for installing packages?

Reply:
Yes, it creates custom repos in this test using `settings.repos[f'rhel{rhelver}_os']`

Reply:
Could you add proxy configure step here?